ui/partials/perf/alertsctrl.html

```diff
@@ -107,6 +107,17 @@
         </div>
       </div>
     </div>
+    <div class="drawer">
+      <div class="collapse" id="alertSummaryNotes">
+        <textarea class="form-control" ng-model="alertSummary.notes" ng-change="alertSummary.editingNotes()">{{alertSummary.notes}}</textarea>
+        <button type="button" class="btn btn-light-bordered" ng-disabled="!alertSummary.notesChanged" ng-click="alertSummary.saveNotes()">Save</button>
+        <button type="button" class="btn btn-light-bordered" ng-disabled="!alertSummary.notesChanged" ng-click="alertSummary.undoNotes()">Undo</button>
```

Comment:
I think I might propose a slightly different design here. I would suggest just showing the notes by the default here in a span (i.e. no textarea) then adding an edit button that pops up a full modal and lets the sheriff modify stuff if they desire. Something like this design from google keep:

Unmodified note (view only):

![image](https://user-images.githubusercontent.com/20569/37526493-e4c7c610-2905-11e8-855e-51f787951cd8.png)

Editing in modal mode:

![image](https://user-images.githubusercontent.com/20569/37526514-f4f0011a-2905-11e8-82c0-4cf0b27b9dc8.png)

We already do something like this for other alert properties, as I recall.

Reply:
@wlach I addressed the small *.js changes you asked for. I really like your proposal with using Google Keep's approach, for editing the notes.
Still, I want to land these changes, to get things going.
I will file a separate bug and address your request as soon as possible.
Are you ok with that?